src/Containers/Microsoft.NET.Build.Containers/Tasks/CreateNewImage.Interface.cs

```diff
@@ -48,7 +48,6 @@ partial class CreateNewImage
     /// <summary>
     /// The kind of local registry to use, if any.
     /// </summary>
-    [Required]
```

Comment:
I missed removing this attribute in https://github.com/dotnet/sdk/pull/32751 which causes the IntegrationTests to fail.